### DISCUSSION TITLE
Safe to upgrade macOS?

### DISCUSSION THREAD

**Discussion by KrisCris** [osy86/HaC-Mini#576] [General]
any anyone had upgraded to 11.3.1?

**Comment by S1999U2** [comment]
Yes, works flawlessly

**Reply by KrisCris** [reply]
thanks!

**Reply by Medizinmann-MD** [reply]
> Yes, works flawlessly

What release version of HaCMini.pkg do you use?

**Reply by KrisCris** [reply]
> > Yes, works flawlessly
> 
> What release version of HaCMini.pkg do you use?

I use the latest one (3.0.1).

**Reply by S1999U2** [reply]
Me too
